k8s/deploy/xebialabs/deployment.yaml.tmpl

```diff
@@ -0,0 +1,33 @@
+{{$app := .AppName | kebabcase}}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{$app}}
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: {{$app}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{$app}}
+spec:
+  replicas: {{.AppReplicas}}
+  selector:
+    matchLabels:
+      app: {{$app}}
+  template:
+    metadata:
+      labels:
+        app: {{$app}}
+    spec:
+      containers:
+      - name: {{$app}}
+        image: xebialabsunsupported/rest-o-rant-api
```

Comment:
If you do provide a `deployment.yaml` file, wouldn't you want to ask the user which container to deploy and which port to expose?

And shouldn't it be in another directory than the `xebialabs` directory? Now it's hard to distinguish from the XL YAML files because they have the same extension.

Reply:
Right, I'll have to input a path to their deployment.yaml